analyses/molecular-subtyping-MB/results/mb-rna-methyl-subtype-concordance.tsv

```diff
@@ -0,0 +1,14 @@
+Methylation Subtype	MB, Group3	MB, Group4	MB, SHH	MB, WNT
```

Comment:
Can we make the headers cleaner so this table can be directly plugged into the paper? Maybe "Group 3 (RNA-Seq)", etc?